getflywheel/local-components
# [LOC-5409] Update navigation design

Component tweaks to match [new Figma designs](https://www.figma.com/file/83UXFJyJBu2JFtgJT7mBWX/%F0%9F%9A%80-Local-Finals?node-id=13127%3A1508&mode=dev) for https://github.com/getflywheel/flywheel-local/pull/1821.

Also changes the z-index of:
- The Popup bubble, so that it appears above the re-styled user profile nav icon, which now has a wider surround in its active state.
- The Downloader overlay, so that the product drawer appears above the overlay when pulling from a site or switching PHP version (the PHP version switcher overlay re-uses the downloader overlay for mysterious reasons). This isn't related to the nav directly but seemed too small a tweak to warrant a fresh pair of PRs.


## To test
Either try it in the app by using https://github.com/getflywheel/flywheel-local/pull/1821, or in this branch:

```
yarn && yarn start
```

Then visit http://localhost:6006/?path=/docs/modules-verticalnav--docs&globals=theme:split to see new styling.
